code/extensions/UserDefinedForm_ControllerAttachmentFilter.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+/**
+ * Fixes security risk of emailing attachments to publicly submitted email addresses
+ *
+ * @author stephen
+ */
+class UserDefinedForm_ControllerAttachmentFilter extends DataExtension {
```

Comment:
Just one more thing, sorry @stephenmcm. Can you please add `Extension` into the class name (and file name) as previously discussed?